### PR TITLE
step 0.13.3

### PR DESCRIPTION
**Pull request by dopey**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by dopey** [comment]
@fxcoudert is there a problem with this build? Seems like it's been in limbo longer than usual.

**Comment by SMillerDev** [comment]
@Brewtestbot test this please

File: Formula/step.rb [hunk 1]
```diff
@@ -15,8 +15,8 @@ class Step < Formula
   depends_on "go" => :build
 
   resource "certificates" do
-    url "https://github.com/smallstep/certificates/releases/download/v0.13.2/step-certificates_0.13.2.tar.gz"
-    sha256 "433e0c040a7161ae767db959528c80db365f74401da0338ef28aeaf748267f06"
+    url "https://github.com/smallstep/certificates/releases/download/v0.13.3/step-certificates_0.13.3.tar.gz"
+    sha256 "a198e0853f36c775392fbb37da1e986e66572270800413dd70d76b31a963158c<Paste>"
```
Comment (alanctkc) [inline Formula/step.rb]: @dopey It looks like the sha is failing in the build. Accidental `<Paste>` text?

```
00:21:56 Warning: Resource certificates reports different SHA256: a198e0853f36c775392fbb37da1e986e66572270800413dd70d76b31a963158c<Paste>
00:21:56 Downloaded to: /Users/brew/Library/Caches/Homebrew/downloads/7aa9f884b7896b20b97a972883342063eb9d7ca18a81494940ac3b8854126461--step-certificates_0.13.3.tar.gz
00:21:56 SHA256: a198e0853f36c775392fbb37da1e986e66572270800413dd70d76b31a963158c
```

https://jenkins.brew.sh/job/Homebrew%20Core%20Pull%20Requests/version=catalina/49015/console

Comment (dopey) [reply Formula/step.rb]: fixed. oof

**Comment by dopey** [comment]
@BrewTestBot test this please

**Comment by dopey** [comment]
@SMillerDev anything special I need to do to have the PR pulled in by brewtestbot? I don't see it in the build pipeline on jenkins. And I don't think it ran automatically when I originally pushed the PR, until you commented asking the brew test bot to test it.

**Comment by fxcoudert** [comment]
We're having some instability in testing, as we recently migrated to a new solution. Please be patient, we're relaunching this that drop out of the queue.

**Comment by dopey** [comment]
@fxcoudert ah, thanks for letting me know. I wasn't sure if there was something I was doing wrong  on my end. Appreciate the update.

**Pull request closed without merging**